client/core/bookie.go

```diff
@@ -876,6 +876,46 @@ func (dc *dexConnection) refreshServerConfig() error {
 	return nil
 }
 
+// subPriceFeed subscribes to the price_feed notification feed and primes the
+// initial prices.
+func (dc *dexConnection) subPriceFeed() {
+	var spots map[string]*msgjson.Spot
+	err := sendRequest(dc.WsConn, msgjson.PriceFeedRoute, nil, &spots, DefaultResponseTimeout)
+	if err != nil {
+		var msgErr *msgjson.Error
+		// Ignore old servers' errors.
+		if !errors.As(err, &msgErr) || msgErr.Code != msgjson.UnknownMessageType {
+			dc.log.Errorf("unable to fetch market overview: %w", err)
+		}
+		return
+	}
+
+	dc.spotsMtx.Lock()
+	dc.spots = spots
```

Comment:
Thinking we should not assign if `spots` is still `nil` after unmarshalling the response.  Server's response could be valid and result in `spots` staying nil if response payload is json `null` I think.
Would want to avoid panic assigning to nil map in `handlePriceUpdateNote`.